places-solution-integrations/analytics/places-trigger.md

```diff
@@ -0,0 +1,68 @@
+# Using Places Events to Send Data to Analytics via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to send data to Adobe Analytics. With your desired property selected in Launch, you can create this rule by completing four steps:
+
+- [Create a new `Rule`](#create-a-rule)
+- [Select the `Event` you wish to trigger the rule](#select-an-event)
+- [Add any other `Conditions` required to trigger the action (optional)](#add-conditions)
+- [Select the `Action` to send data to Analytics and define your payload](#define-the-action)
+
+## Create a Rule
+
+In the `Rules` tab, click the `Create New Rule` button.
+
+If you don't have existing rules for this property, the button will be in the middle of the screen:
+
+<img src="../../.gitbook/assets/integrations/addFirstRule.png" width="800" /><br />
+
+If your property already has rules, the button will be in the top right of the screen:
+
+<img src="../../.gitbook/assets/integrations/addRuleFromList.png" width="800" /><br />
+
+## Select an Event
+
+First, you can name your rule. We've named this rule "Send Data to Analytics".
+
+Next, click the "Add" button under the "Events" section:
+
+<img src="../../.gitbook/assets/integrations/analytics/pt-addEvent.png" width="800" /><br />
+
+Select `Places` (or `Places - Beta`) from the "Extension" dropdown, then select `Enter POI` from the "Event Type" dropdown. Hit the "Keep Changes" button.
```

Comment:
Step 3, 4, and 5:
3. From the **Extension** drop-down list, select **Places** (or **Places - Beta**) .
4. From the **Event Type** drop-down list, select **Enter POI** . 
5. Click **Keep Changes**.